### PR TITLE
Update ruleset to be similar to EVO 2019,

### PR DESCRIPTION
**Pull request by metalvegetarianoprogresivo**
## What does this do?
We updated the ruleset so this is similar to the one for EVO 2019, since most of the local (and national) communities for SSBU will be using this one. This is open for discussion, comment on this PR if the changes are good for you or if you have any questions/proposals, feel free to comment on this PR. We'll update it so everyone on the league is good with this ruleset.

We are using the EVO 2019 rules, you can [check it here](https://evo.shoryuken.com/additional-rules/).

## How can this change be undone in case of failure?

1. Notify the administrators
2. Request a revert of the PR

ping @Xotl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,15 +124,32 @@ Couching during an in progress game is not allowed. Please stick to the rules.
...
+- Score Display: Off.
+- % Show Damage: Yes.
+- Custom Balance: Off.
+- Echo Fighters: Separate.
+- Radar: Big.
+- Allowed Controllers: Nintendo GameCube, Nintendo Pro Controller, Nintendo Joy-Con, SmashBox.
+- Mii Fighters: All moveset combinations are legal.
+- Game one striking: (p1 - p2 - p2 - p1). In case both players don’t want to strike, the default starter stage will be **Pokémon Stadium 2**. You may not play on a stage that is not on the available stages list. This will result in a double disqualification/penalization.
 
 If case of time out, the winner will be decided by the winner screen. In case of a tie, the winner will be the player with less damage at the end of the match. In case of a tie of damage or double KO, the players will play a one stock 3 minutes round on the same stage to decide the winner. In case of a player SD, the winner will be decided by the winner screen.
```
Comment (Xotl) [inline README.md]: shouldn't that line start with `In case of time out` instead of `If case of time out`?.

Comment (metalvegetarianoprogresivo) [reply README.md]: Sounds good, I'll change it.

**Comment by Jarv1193** [comment]
LGTM

**Comment by leumashz** [comment]
LGTM

**Pull request merged**